javascript/chronometer.js

```diff
@@ -1,26 +1,35 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
+
   startClick(callback) {
-    // ... your code goes here
+    setInterval(
+      () => this.currentTime++, 1000);
   }
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60)
   }
   getSeconds() {
-    // ... your code goes here
+    return (this.currentTime - Math.floor(this.getMinutes() * 60))
   }
-  twoDigitsNumber() {
-    // ... your code goes here
+  twoDigitsNumber(number) {
+    if (number < 10) {
```

Comment:
This function should return a string. The function name as mentioned in the iteration is therefore confusing. It would be better named something like "getTwoDigitsString".

The code could be minimized as follows:
return (number < 10 ? '0' : '') + number;